doctr/utils/data.py

```diff
@@ -23,6 +23,7 @@
 # matches bfd8deac from resnet18-bfd8deac.ckpt
 HASH_REGEX = re.compile(r'-([a-f0-9]*)\.')
 USER_AGENT = "mindee/doctr"
+DOCTR_CACHE_DIR_ENV_NAME = "DOCTR_CACHE_DIR"
```

Comment:
I think we can remove this if there is no path to an existing folder as DOCTR_CACHE_DIR env variable set than it will use our default or we do:

`DOCTR_CACHE_DIR_PATH = os.environ.get(DOCTR_CACHE_DIR, os.path.join("~", ".cache", "doctr"))`

Reply:
I agree :+1: